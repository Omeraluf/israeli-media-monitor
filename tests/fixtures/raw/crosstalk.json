[
  {
    "title": "כתבה א",
    "summary": "טקסט",
    "url": "https://ex.com/a",
    "source": "x",
    "published_iso": "2025-08-10T10:00:00",
    "published_dt": "2025-08-10T10:00:00",
    "clean_title": "כתבה א",
    "dedup_key": "כתבה א_2025-08-10"
  },
  {
    "title": "כתבה ב",
    "summary": "עוד",
    "url": "https://ex.com/b",
    "source": "x",
    "published_iso": "2025-08-10T10:10:00",
    "published_dt": "2025-08-10T10:10:00",
    "clean_title": "כתבה ב",
    "dedup_key": "כתבה ב_2025-08-10"
  }
]
