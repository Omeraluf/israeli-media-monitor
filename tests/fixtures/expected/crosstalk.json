[
  {
    "title": "עדכון",
    "summary": "פרטים יימסרו בהמשך.",
    "url": "https://ex.com/u1",
    "source": "x",
    "published_iso": "2025-08-10T12:00:00",
    "published_dt": "2025-08-10T12:00:00",
    "clean_title": "עדכון",
    "dedup_key": "עדכון_2025-08-10"
  },
  {
    "title": "ידיעה",
    "summary": "מידע נוסף בקרוב.",
    "url": "https://ex.com/u2",
    "source": "x",
    "published_iso": "2025-08-10T12:05:00",
    "published_dt": "2025-08-10T12:05:00",
    "clean_title": "ידיעה",
    "dedup_key": "ידיעה_2025-08-10"
  }
]
